Fatal error: Call to undefined function i18n_sync_node_delete_submit() : https://www.drupal.org/node/2209973
--- i18n_sync/i18n_sync.module
+++ i18n_sync/i18n_sync.module
@@ -74,15 +74,6 @@ function i18n_sync_field_info_alter(&$fields) {
 /**
  * Implements hook_form_FORM_ID_alter().
  */
-function i18n_sync_form_node_admin_content_alter(&$form, &$form_state) {
-  if (!empty($form['operation']) && $form['operation']['#value'] == 'delete') {
-    $form['#submit'] = array_merge(array('i18n_sync_node_delete_submit'), $form['#submit']);
-  }
-}
-
-/**
- * Implements hook_form_FORM_ID_alter().
- */
 function i18n_sync_form_node_type_form_alter(&$form, &$form_state) {
   if (isset($form['type'])) {
     $type = $form['#node_type']->type;
